tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,44 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe.only('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
```

Comment:
```suggestion
	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
```

Reply:
Was WIP test, added to discuss the is-of mapping work.